extension/src/popup/components/sendPayment/SendSettings/TransactionFee/index.tsx

```diff
@@ -63,26 +63,29 @@ export const SendSettingsFee = () => {
               <FormRows>
                 <Field name="transactionFee">
                   {({ field }: FieldProps) => (
-                    <Input
-                      id="transaction-fee-input"
-                      className="SendTo__input"
-                      type="number"
-                      {...field}
-                    ></Input>
+                    <>
+                      <Input
+                        id="transaction-fee-input"
+                        className="SendTo__input"
+                        type="number"
+                        {...field}
+                      ></Input>
```

Comment:
So so so NIT: but you can do `<Input />` if the element has no children :)

Reply:
good 👁️  https://github.com/stellar/freighter/pull/503/commits/31d1c4285f29f4cafc3e2065d7f5977bc1ce4024